lib/travis/api/v3/routes.rb

```diff
@@ -27,6 +27,15 @@ module Routes
       end
     end
 
+
+    resource :cron do
+      capture id: :digit
+      route '/cron/{cron.id}'
+      get :find
+
+      post :delete, '/delete'
```

Comment:
@joshk we are somehow afraid that the user accidentally deletes a cron due to confusion of POST and GET. Especially since there is no further JSON needed in the POST request (like it is in `create`).